obal/data/playbooks/bump-release.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: packages
+  serial: 1
+  gather_facts: no
+  roles:
+    - package_variables
+  tasks:
+    - name: 'Bump release'
+      command: "rpmdev-bumpspec {{ inventory_hostname }}.spec"
```

Comment:
given we have repos where the package is `tfm-foo`, but the spec is `foo.spec`, should that be a `shell` with `*.spec` instead?

Reply:
It did not like *.spec :/ I'll add spec file detection logic. Stupid f-packaging ;)

Reply:
for `*.spec`, it needs `shell` instead of `command`, but yeah, `find` is good too :+1:

Reply:
@ehelms pending anwer to the above question about the full path instead of the chdir ;)